### ISSUE TITLE
Gauge disappeared in 1.4.4.0

### ISSUE THREAD

**Issue by goller** [influxdata/chronograf#3200]
<img width="929" alt="image" src="https://user-images.githubusercontent.com/1922921/38781023-da454724-40a4-11e8-9dc1-1329de44ac93.png">

In graph "Number of Nodes"

**Comment by kevinjos** [comment]
I'm getting the same error since updating to 1.4.4.0. It's also worth noting that the downstream consequence is that additional dashboard widgets fails to render.

**Comment by goller** [comment]
@kevinjos Ah that makes sense... I was just about to report another problem, but, I bet it is because downstream widgets don't render.

**Comment by Lootvoet** [comment]
Same issue. Even when downgrading back to 1.4.3.3.

**Comment by russorat** [comment]
Thanks for the reports everyone. We will take a look and address this today.

**Comment by lukevmorris** [comment]
@Lootvoet When downgrading from 1.4.4.0 -> 1.4.3.3, you might also try replacing your bolt database (generally `chronograf-v1.db`) with the automatic backup taken at the beginning of the upgrade process. By default it will be in the `backup/` directory.

**Comment by lukevmorris** [comment]
@Lootvoet @goller @kevinjos Thanks for bringing this to our attention. We've pushed a fix, which is now generally available in Chronograf 1.4.4.1.

This required a change to an existing migration file (which ran automatically when you moved from 1.4.3.x -> 1.4.4.0). Because this migration has already been marked as run, resolving this issue on your side is *not as simple as upgrading* to 1.4.4.1. You will first need to restore from a backup of your internal Chronograf database (i.e. your boltdb). The process is as follows:


1. Locate your Chronograf database (`chronograf-v1.db`) and your backup directory (`backup/`). They should be siblings of each other, and they live in your specified `--bolt-path`. By default, this is the current working directory where you execute the `chronograf` binary.
1. Inside your backup directory, you should see a database file with a suffix corresponding with the version you upgraded from. For example, if you moved `1.4.3.3` -> `1.4.4.0`, you should find a file called `backup/chronograf-v1.db.1.4.3.3`.
1. Stop your chronograf server
1. Remove your "current" database, and replace it with the backup:
```
rm chronograf-v1.db
mv backup/chronograf-v1.db.1.4.3.3 chronograf-v1.db
```
5. Upgrade to the latest chronograf (`1.4.4.1`): https://portal.influxdata.com/downloads
6. Start the chronograf server. This will automatically run the updated migration against your most-recent dashboard configuration.


If you have any questions about this process, it's likely that I need to clarify further -- please let me know. Sorry for the trouble!
